### PR TITLE
URL -> url in POST action

### PR DESCRIPTION
**Pull request by crbrox**
Fixes issue #45

@fgalan 
@dmoranj 
@iariasleon

### CODE REVIEW

File: lib/models/postAction.js [hunk 1]
```diff
@@ -26,15 +26,16 @@ var myutils = require('../myutils'),
     logger = require('logops');
```
Comment (fgalan) [inline lib/models/postAction.js]: CHANGES_NEXT_RELEASE entry missing.

Comment (crbrox) [inline lib/models/postAction.js]: Fixed in 6879b34

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj to final LGTM + merge)

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**